app/Filament/Resources/CollaboratorsResource/RelationManagers/GithubReposRelationManager.php

```diff
@@ -31,20 +31,21 @@ public static function table(Table $table): Table
         return $table
             ->columns([
                 Tables\Columns\TextColumn::make('name'),
+                
             ])
             ->filters([
                 //
             ])
             ->headerActions([
-                // Tables\Actions\CreateAction::make(),
+                Tables\Actions\CreateAction::make(),
             ])
             ->actions([
-                // Tables\Actions\EditAction::make(),
-                // Tables\Actions\ViewAction::make(),
-                // Tables\Actions\DeleteAction::make(),
+                Tables\Actions\EditAction::make(),
+                Tables\Actions\ViewAction::make(),
+                Tables\Actions\DeleteAction::make(),
             ])
             ->bulkActions([
-                // Tables\Actions\DeleteBulkAction::make(),
+                Tables\Actions\DeleteBulkAction::make(),
```

Comment:
why are we enabling these features?

might be commented on because these informations are scanned and collected from GitHub, we do not allow them to be modified, but if you have any specific reason to allow it, please share.

Reply:
ok, I will comment on it.